frontend/app/mocks/db.ts

```diff
@@ -40,10 +40,11 @@ const db = factory({
   },
   letter: {
     id: primaryKey(faker.string.uuid),
-    issuedOn: () => faker.date.past({ years: 1 }).toISOString().split('T')[0],
-    letterType: oneOf('letterType'),
     referenceId: String,
+    dateSent: () => faker.date.past({ years: 1 }).toISOString().split('T')[0],
```

Comment:
`issuedOn` is the same as `dateSent`. Remove dateSent and keep `issuedOn`.

Reply:
Fixed: 04e0b73a6520ee0dc46d6b68ece420e1df40cca8